src/runner.js

```diff
@@ -54,6 +54,26 @@ export function run(stdout, stdin, stderr, argv) {
     stdin.fd
   );
 
+  const issues = configuration.validate();
+
+  if (issues.length > 0) {
+    issues.map(issue => {
+      var prefix;
+      if (issue.type == 'error') {
+        prefix = `${chalk.red(figures.cross)} Error`;
+      } else {
+        prefix = `${chalk.yellow(figures.warning)} Warning`;
+      }
+      stderr.write(
+        `${prefix} on ${chalk.bold(issue.field)}: ${issue.message}\n\n`
+      );
+    });
+
+    if (issues.some(issue => issue.type == 'error')) {
+      return 1;
```

Comment:
At the moment, `0` = success, `1` = linter errors, `2` = uncaught exception, but I wonder if we should include configuration errors in `2`. To help differentiate between both failures.

Any thoughts?

Reply:
Yeah, I think that would be useful.